# Image Data Annotation
A data management study file with a collection of experiments for microscopic data 
generated and tested by MRI using OMERO & FAIR principles

# Authors
Quentin Szabo,  Axelle Donjon, Ivana Jerković , Giorgio L. Papadopoulos, Thierry Cheutin, Boyan Bonev , Elphège P. Nora , 
                          Benoit G. Bruneau , Frédéric Bantignies , and Giacomo Cavalli

# Description of The Project
The genome folds into a hierarchy of three-dimensional structures within the nucleus. At the sub-megabase scale, chromosomes form topologicallyassociating domains (TADs)1–4. However, how TADs fold in single cells is elusive. Here, we reveal TAD features inaccessible to cell population analysis by using super-resolution microscopy.
TAD structures and physical insulation associated with their borders are variable between individual cells, yet chromatin intermingling is enriched withi TADs compared to adjacent TADs in most cells. The spatial segregation of TADs is further exacerbated during cell differentiation. Favored interactions within TADs are regulated by cohesin and CTCF through distinct mechanisms: cohesin generates chromatin contacts and intermingling while CTCF prevents inter-TAD contacts. Furthermore, TADs are subdivided into discrete nanodomains, which persist in cells depleted of CTCF or cohesin, whereas disruption of nucleosome contacts alters their structural organization. 
Altogether, these results provide a physical basis for the folding of individual chromosomes at the nanoscale.
